packages/cli/commands/accounts/use.js

```diff
@@ -5,47 +5,24 @@ const {
   updateDefaultAccount,
   getAccountId: getAccountIdFromConfig,
 } = require('@hubspot/cli-lib/lib/config');
-const { loadAndValidateOptions } = require('../../lib/validation');
 
-const { getAccountId } = require('../../lib/commonOpts');
 const { trackCommandUsage } = require('../../lib/usageTracking');
-const { promptUser } = require('../../lib/prompts/promptUtils');
 const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { selectAccountFromConfig } = require('../../lib/prompts/accountsPrompt');
+const { loadAndValidateOptions } = require('../../lib/validation');
 
 const i18nKey = 'cli.commands.accounts.subcommands.use';
 
-const selectAccountFromConfig = async config => {
-  const { default: selectedDefault } = await promptUser([
-    {
-      type: 'list',
-      look: false,
-      name: 'default',
-      pageSize: 20,
-      message: i18n(`${i18nKey}.promptMessage`),
-      choices: config.portals.map(p => ({
-        name: `${p.name} (${p.portalId})`,
-        value: p.name || p.portalId,
-      })),
-      default: config.defaultPortal,
-    },
-  ]);
-
-  return selectedDefault;
-};
-
 exports.command = 'use [--account]';
 exports.describe = i18n(`${i18nKey}.describe`);
 
 exports.handler = async options => {
-  await loadAndValidateOptions({ debug: options.debug });
+  await loadAndValidateOptions(options, false);
```

Comment:
Updated to use the full options object. Before, it was never receiving the `--account` flag if passed